### PR TITLE
Patch-3

### PR DESCRIPTION
**Pull request by TurtleGod7**
I revised some logic and made the code a little 🤏more readable. Also, I changed the README to give more info to others viewing this repository which coincides with the pull request #5. I'm a little lazy right now but maybe we should fill out the contributing guidelines? I added it in reference in the new README and you can check it out by going to **Insights**, then **Community Standards**.

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -142,6 +148,7 @@ void initialize() {
             pros::delay(10);
         }
     });
+	// the color sort task can't be put here due to some issue with defining the color signatures?
```
Comment (techsupport101) [inline src/main.cpp:151]: define the sigs (237 ish) at the top after the constructors so you can put the color sort task up here

**Review by techsupport101 — COMMENTED** [review]
see comments. almost there

File: src/main.cpp [hunk 2]
```diff
@@ -40,7 +40,7 @@ pros::Controller ctrl (CONTROLLER_MASTER);
 
 //lcd stuffs
 void on_left_button() {
-    if(sortedColor <= 2 && sortedColor >= 0) {
+    if(0 <= sortedColor && sortedColor < 2) {
```
[the remark this answers is not in the document]
Comment (TurtleGod7) [reply src/main.cpp:43]: No because it 2 is inputted, 3 will be returned which is out of range

File: src/main.cpp (hunk 1 above)
Comment (TurtleGod7) [reply src/main.cpp:151]: Yeah I tried that and I got an error saying that it can't hold the value or something?

Comment (techsupport101) [reply src/main.cpp:151]: hm, I'll see what I can figure out

**Review by techsupport101 — APPROVED** [review]
good enough lmao

**Pull request merged**